shared/panels/issue-sidepanel/_components/apply-panel/apply-panel.tsx

```
@@ -0,0 +1,123 @@
+
+            <TypographyH3>Apply and Wait for Assignment</TypographyH3>
+
+            <TypographyMuted>
+              Perfect for contributors who love structure and value support from the maintainer.
+            </TypographyMuted>
+
+            <TypographyP>
+              ✅ Clear direction from the maintainer
+              <br />
+              ✅ Structured support throughout
+              <br />✅ Ensure you're on the right track before diving in
```

ESLint

[error] `'` can be escaped with `&apos;`, `&lsquo;`, `&#39;`, `&rsquo;`.

(react/no-unescaped-entities)